CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# Release 0.32.2
+
+## What's New
+
+* Terminator performance improvements
+* API Rate Limiter enabled by default
```

Comment:
This is really great to see. It's a significant boost in the default security posture.